A32NX/html_ui/Pages/A32NX_Core/A32NX_GroundReference.js

```diff
@@ -0,0 +1,12 @@
+class A32NX_GroundReference {
+    constructor() {
+        console.log('A32NX_GroundReference constructed');
+    }
+    init() {
+        console.log('A32NX_GroundReference init');
+    }
```

Comment:
Are the console log statements supposed to end up in master?

Reply:
I followed the scheme of the other modules and logs are present.
PR has already been revised and ready to test, I think they must be present at the moment in that part...